jdk/src/share/native/com/sun/java/util/jar/pack/unpack.cpp

```diff
@@ -3746,7 +3757,7 @@ char* entry::string() {
 void print_cp_entry(int i) {
   entry& e = debug_u->cp.entries[i];
   char buf[30];
-  sprintf(buf, ((uint)e.tag < CONSTANT_Limit)? TAG_NAME[e.tag]: "%d", e.tag);
+  snprintf(buf, sizeof(buf), ((uint)e.tag < CONSTANT_Limit)? TAG_NAME[e.tag]: "%d", e.tag);
   printf(" %d\t%s %s\n", i, buf, e.string());
```

Comment:
Rather than continue to use `snprintf()`, perhaps we should apply a better change from upstream; see
* [8074839: Resolve disabled warnings for libunpack and the unpack200 binary](https://github.com/ibmruntimes/openj9-openjdk-jdk11/commit/4ffef60c12fa61a5157b62cccc04e1a0d002f825)

Reply:
Updated.